scripts/start_parallel_jobs_slurm.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/packages/python/anaconda/2.1.0/bin/python
```

Comment:
Why was this change made? This needs to be reverted as it's not a path that most users would have on their system.